x-pack/elastic-agent/Jenkinsfile.yml

```diff
@@ -95,10 +95,10 @@ stages:
         e2e:
             enabled: false
         stage: packaging
-    packaging-arm:
-        packaging-arm: "mage package"
-        e2e:
-            enabled: false
-        platforms:             ## override default label in this specific stage.
-          - "arm"
-        stage: packaging
+#    packaging-arm: See https://github.com/elastic/beats/issues/26239
```

Comment:
The async pipeline that triggers the packaging might need also the same change:
- https://github.com/elastic/beats/blob/master/.ci/packaging.groovy#L211

Reply:
Done, thanks!